app/src/main/java/io/github/droidkaigi/confsched2018/data/repository/ContributorDataRepository.kt

```diff
@@ -0,0 +1,39 @@
+package io.github.droidkaigi.confsched2018.data.repository
+
+import io.github.droidkaigi.confsched2018.data.api.GithubApi
+import io.github.droidkaigi.confsched2018.data.db.ContributorDatabase
+import io.github.droidkaigi.confsched2018.data.db.entity.mapper.toContributors
+import io.github.droidkaigi.confsched2018.model.Contributor
+import io.github.droidkaigi.confsched2018.util.rx.SchedulerProvider
+import io.reactivex.Completable
+import io.reactivex.Flowable
+import timber.log.Timber
+import javax.inject.Inject
+
+class ContributorDataRepository @Inject constructor(
+        private val api: GithubApi,
+        private val contributorDatabase: ContributorDatabase,
+        private val schedulerProvider: SchedulerProvider
+) : ContributorRepository {
+    override fun loadContributors(): Completable {
+        return api.getContributors(OWNER, REPO, MAX_PER_PAGE)
+                .doOnSuccess {
+                    if (DEBUG) it.forEach { Timber.d("$it") }
```

Comment:
It's okay for now!
Currently we setup Timber like this.

```kotlin
Timber.plant(Timber.DebugTree())
```

But it would be better to create Tree for production which ignores the debug log 📝